tests/acceptance/features/cliCommands/uploadSessions.feature

```diff
@@ -83,3 +83,27 @@ Feature: List upload sessions via CLI command
       | file3.txt |
     And the CLI response should not contain these entries:
       | file1.txt |
+
+
+  Scenario: user restarts upload sessions that are postprocessing
```

Comment:
```suggestion
  Scenario: user restarts upload sessions that are postprocessing
```
Can we write `admin user` here for more clarity?

Reply:
Other scenarios for CLI commands have the scenario title as `restart upload sessions that are processing`. I will update in similar way to make things consistent for cli command tests. :+1: